BRAVE/docker-compose.prod.yml

```diff
@@ -0,0 +1,60 @@
+services:
+  django-prod:
+    build: ./backend
+    container_name: django-prod
+    command: python manage.py runserver 0.0.0.0:8000
+    ports:
+      - 8001:8000
+    env_file:
+      - ./.env.dev
+      - ./backend/.env
+    volumes:
+      - ./backend/:/srv/app
+      - static:/srv/app/static
+      - media:/srv/app/media
+    depends_on:
+      - db-prod
+
+  react-prod:
+    build: 
+      context: ./frontend
+      dockerfile: Dockerfile.prod
+    container_name: react-prod
+    command: npm start
+    ports:
+      - 3001:3000
```

Comment:
This is a very minor issue, but since port `3001` is currently in use can we please change this to `3000`